asab/zookeeper/container.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, app, config_section_name, config=None, z_path=None):
 
 	async def initialize(self, app):
 		await self.ZooKeeper.start()
-		await self.ZooKeeper.ensure_path(self.ZooKeeperPath)
+		await self.ZooKeeper.ensure_path(self.ZooKeeperPath + "/run")
```

Comment:
Unfortunately, this is not a proper location of this fix.
It should be done in the Asab API Service - that's where it is needed.